draft-irtf-cfrg-hpke.md

```diff
@@ -598,6 +608,11 @@ octet strings for public keys.
 
 [[ TODO ]]
 
+# Message Encoding
+
+This document does not specify a wire format encoding for HPKE messages. Applications
+which adopt HPKE must therefore specify an unambiguous encoding mechanism. 
```

Comment:
Nit: s/which/that/

Reply:
It might be good to specify what they need to encode, namely:

* The encapsulated value `enc`
* Ciphertext value(s), in order if multiple
* Any info values that are not implicit